### ISSUE TITLE
New manage_db.sh script fails to remove python cache due to permissions problem

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy#14678]
@dirowa discovered this on our priv-sep, ansible maintained galaxy

```
(base) galaxy@galaxy:/srv/galaxy/server$ find lib/galaxy/model/migrations/alembic -name '*.pyc' -delete                                                       
find: cannot delete ‘lib/galaxy/model/migrations/alembic/__pycache__/env.cpython-38.pyc’: Permission denied                                                   
find: cannot delete ‘lib/galaxy/model/migrations/alembic/versions_tsi/__pycache__/d4a650f47a3c_create_tsi_branch.cpython-38.pyc’: Permission denied           
find: cannot delete ‘lib/galaxy/model/migrations/alembic/versions_gxy/__pycache__/b182f655505f_add_workflow_source_metadata_column.cpython-38.pyc’: Permission
ed                                                                                                                                                            
find: cannot delete ‘lib/galaxy/model/migrations/alembic/versions_gxy/__pycache__/e7b6dcb09efd_create_gxy_branch.cpython-38.pyc’: Permission denied           
find: cannot delete ‘lib/galaxy/model/migrations/alembic/versions_gxy/__pycache__/186d4835587b_drop_job_state_history_update_time_.cpython-38.pyc’: Permission
ed                                                                                                                                                            
find: cannot delete ‘lib/galaxy/model/migrations/alembic/versions_gxy/__pycache__/6a67bf27e6a6_deferred_data_tables.cpython-38.pyc’: Permission denied  
```

**Comment by mvdbeek** [comment]
Would you be able to delete the .pyc files as a task in ansible ? @jdavcs can we skip the *.pyc delete ? I have a vague feeling we did that for sqlalchemy-migrate, which would find migrations based on the file name, so when you switched branches to an older release it would still think the migration is present.

**Comment by hexylena** [comment]
Yes, absolutely could be in ansible. That would be ideal.

**Comment by jdavcs** [comment]
This should be fixed in #14537. Please feel free to reopen if this reoccurs.
